app/views/projects/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="w-50 mx-auto">
+  <%= simple_form_for(@project) do |f| %>
+    <%= f.input :project_name %>
+    <%= f.input :description %>
+    <%= f.input :people_required %>
+    <%= f.input :location %>
+    <%= f.input :start_date %>
+    <%= f.input :end_date %>
+    <%= f.input :reward_earned %>
+    <%= f.input :points_earned %>
+    <%= f.submit class: "btn btn-primary" %>
```

Comment:
is this the correct use of f.submit? I thought you only could add a label to this, but you added a class to it? Sorry perhaps I don't understand this enough

Reply:
![Screenshot (36)](https://user-images.githubusercontent.com/83071733/151156390-72f3d24e-7b51-435a-8944-b5a368a3e43b.png)
that is for this beautiful blue button

Reply:
OK thank you! Looks good. I didn't know you could do that